README.md

```diff
@@ -1,12 +1,18 @@
-## Cloud.gov PHP Example Application:  Wordpress
+## Cloud.gov PHP Example Application:  WordPress
 
 This is an example application which can be run on Cloud.gov using the CloudFoundry [PHP Build Pack].
```

Comment:
We're missing a reference to http://docs.cloudfoundry.org/buildpacks/php/index.html

Reply:
✅